tests/test0000.d

```diff
@@ -1,3 +1,5 @@
+// RUN: %sdc %s -o %t && sh -c '%t; [ $? = 42 ]'
```

Comment:
I don't think this is the right way forward. We should use this to check the codegen for LLVM, but I'd like to see all this test suite ran by runner.d using libd as a library rather than running sdc again and again.